src/main/groovy/util/Node.java

```diff
@@ -537,26 +539,40 @@ private NodeList getByName(String name) {
 
     /**
      * Provides a collection of all the nodes in the tree
-     * using a depth first traversal.
+     * using a depth-first preorder traversal.
      *
      * @return the list of (depth-first) ordered nodes
      */
     public List depthFirst() {
+        return depthFirst(true);
+    }
+
+    /**
+     * Provides a collection of all the nodes in the tree
+     * using a depth-first traversal.
+     *
+     * @param preorder if false, a postorder depth-first traversal will be performed
```

Comment:
As mentioned in the Jira, I am not aware of any widely used inorder traversal algorithms for non-binary trees. I have followed some of the computer science literature/research in that area but not all. Do you have references to such an algorithm?